src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource.cs

```diff
@@ -92,7 +90,6 @@ private sealed partial class TagSource : ForegroundThreadAffinitizedObject
             private readonly CancellationTokenSource _initialComputationCancellationTokenSource = new();
 
             public TaggerDelay AddedTagNotificationDelay => _dataSource.AddedTagNotificationDelay;
-            public TaggerDelay RemovedTagNotificationDelay => _dataSource.RemovedTagNotificationDelay;
```

Comment:
both these members always had the same value provided (and never overridden). So they could be removed, and hteir value inlined into the locations where they were previously read.